TDL12-Python-Lambda-Cloudwatch/terraform-code/_provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  region  = var.aws_region
+  profile = "default"
```

Comment:
It is better not to set up a profile here for the below reasons:
- It indicates the default profile is configured which is not always the case
- On CI/CD tools such as Jenkins,  the best practice is to avoid IAM profile 
- Leaving AWS credentials to end-users will give PPL more flexibility.